healthtools/documents.py

```diff
@@ -16,28 +16,50 @@
                      'RN', 'Registered Nurse'],
         'sms_field': 'name'
     },
-    'nhif': {
-        'search_type': 'elastic',
-        'keywords': ['nhif', 'bima', 'insurance', 'insurance fund',
-                     'health insurance', 'hospital fund'],
-        'sms_field': 'name'
-    },
     'health-facilities': {
         'search_type': 'elastic',
         'keywords': ['hf', 'hospital', 'dispensary', 'clinic', 'hospitali',
                      'sanatorium', 'health centre'],
         'sms_field': 'name'
     },
+    'nhif-inpatient': {
+        'search_type': 'elastic',
+        'keywords': ['inpatient','nhif-inpatient','nhif inpatient'
+                     'bima-inpatient','bima inpatient', 
+                     'inpatient insurance', ' inpatient insurance fund', 
+                     'inpatient health insurance', 'inpatient hospital fund'],
+        'sms_field': 'name'
+    },
+    'nhif-outpatient': {
+        'search_type': 'elastic',
+        'keywords': ['outpatient', 'nhif-outpatient', 'nhif outpatient'
+                     'outpatient insurance','bima-outpatient',
+                     'bima outpatient', 'outpatient insurance fund',
+                     'outpatient health insurance', 'outpatient hospital fund'],
+        'sms_field': 'name'
+    },
+    'nhif-outpatient-cs': {
+        'search_type': 'elastic',
+        'keywords': ['outpatient-cs','nhif-outpatient-cs', 'nhif outpatient cs', 
+                     'outpatient-cs insurance', 'outpatient cs insurance', 'outpatient-cs insurance fund ',
+                     'outpatient cs insurance fund ', 'outpatient-cs health insurance', 'outpatient cs health insurance',
+                     'outpatient cs hospital fund', 'outpatient-cs hospital fund', 'outpatient cs bima', 
+                     'outpatient cs-bima', 'outpatient cs insurance', 'outpatient-cs insurance', 'outpatient-cs insurance fund',
+                     'outpatient cs insurance fund','outpatient cs health insurance', 'outpatient-cs health insurance', 
+                     'outpatient-cs hospital fund', 'outpatient cs hospital fund'
+                     ],
+        'sms_field': 'name'
+    },
```

Comment:
What was the thought process on the keyword choices?

Reply:
@DavidLemayian i was think of different ways a user would query for the three docs . This doc is open for more ideas as i realize that i may not be able to exhaust all the query matches that can be used.

Reply:
@DavidLemayian yes i am using Pylint. i have rectified the mistakes. I will be more keen next time before doing a PR.

Reply:
@Awinja-Andela What are the example queries someone might send in an SMS?

Reply:
@DavidLemayian it would probably be something like:
nhif in kitale , nhif hospitals in mombasa, mombasa hospital nhif etc
 Then i am assuming they will be prompted to specify the type of nhif they would like to search for, which would lead them to do something like:
nhif inpatient kitale or
nhif outpatient mombasa or 
outpatient nhif kakamega and so on.